examples/abbas_work_area/OOP_contact.py

```diff
@@ -0,0 +1,152 @@
+import json
+import os
+from OOP_contact_record import ContactRecord
+from CSVdatastore import CSVDataStore
+from TXTdatastore import TXTDataStore
+from JSONdatastore import JSONDataStore
+from examples.abbas_work_area.datastore import DataStore
+
+
+class ContactBook:
+    BASE_FILE_PATH = "C:/Users/71519/Documents/genz-todo-list-app/examples/abbas_work_area/data/store_contacts"
```

Comment:
Should we change it to relative path or use something like os.path.abspath(os.path.basedir(__file__))